File: src/client/classes/api/req_res/get-plugin-config-html-response.ts
'use strict'

import { GkillError } from '../gkill-error'
import { GkillMessage } from '../gkill-message'

export class GetPluginConfigHTMLResponse {
    errors: Array<GkillError>
    messages: Array<GkillMessage>
    html: string

    constructor() {
        this.errors = new Array<GkillError>()
        this.messages = new Array<GkillMessage>()
        this.html = ''
    }
}

File: src/server/gkill/api/req_res/get_plugin_config_html_response.go
package req_res

import "github.com/mt3hr/gkill/src/server/gkill/api/message"

type GetPluginConfigHTMLResponse struct {
	Messages message.GkillMessages `json:"messages"`
	Errors   message.GkillErrors   `json:"errors"`
	HTML     string                `json:"html"`
}
